### DISCUSSION TITLE
How to get filepath from VFileInput

### DISCUSSION THREAD

**Discussion by giiyms** [Kitware/trame#194] [Q&A]
Hello,

I need to interact with some large files (10GB+) that I would rather not move to temporary storage.

Is there a way to get the full filepath from VFileInput instead of just the name?
I was thinking of just changing it to textbox but it would be nice to have the ability for the user to interactively select it.

EDIT:
Sorry, evidently crap as searching, was looking for vfileinput but now see a similar question:
https://github.com/Kitware/trame/discussions/191

Basically, I need to point to a Ansys database filepath. 
Ansys Mechanical will open that file directly, run a simulation.
After I will read the results, and display the results in Trame (using Tauri for standalone)
